libr/core/cmd_anal.c

```diff
@@ -47,6 +47,7 @@ static void var_help(RCore *core, char ch) {
 		"afvg", " [idx] [addr]", "define var get reference",
 		"afvs", " [idx] [addr]", "define var set reference",
 		"afx", "[-] [from] [to]", "manipulate function xrefs",
+		"afM", "", "print byte map of functions coverage",
```

Comment:
Isn't it in the wrong subcommand? To show this I have to do `afv?` which should be for variable and argument commands.